services/wallet/leaderboard/config.go

```diff
@@ -0,0 +1,56 @@
+package leaderboard
+
+import (
+	"github.com/status-im/status-go/params"
+)
+
+const (
+	defaultFullDataInterval    = 10 // Default full data refresh interval in seconds
```

Comment:
Instead having a comment here saying it refers to seconds, could we update this and the line below to something like 
```go
const (
	defaultFullDataInterval    = 10 * time.Second
	defaultPriceUpdateInterval = 1 * time.Second
)